### PR TITLE
Fix broken pagination links

### PR DESCRIPTION
**Pull request by cesidio**

### CODE REVIEW

**Review by cpjmcquillan — APPROVED** [review]
Can you add a context quickly to `jobseekers_can_view_all_the_jobs_spec.rb` that tests pagination when coming from the home page? Or we can make a separate PR for that

**Review by david-mears-dfe — CHANGES_REQUESTED** [review]
When I check out this branch locally, I get the same number of results no matter what location category I click.

File: config/routes.rb [hunk 1]
```diff
@@ -151,9 +151,9 @@
   match "/500", to: "errors#internal_server_error", via: :all
 
   # If parameters are used that are the same as those in the search form, pagination with kaminari will break
-  match "teaching-jobs-in-:location",
+  match "teaching-jobs-in-:location_facet",
         to: "vacancies#index", as: :location, via: :get,
-        constraints: ->(request) { LocationPolygon.include?(request.params[:location]) }
+        constraints: ->(request) { LocationPolygon.include?(request.params[:location_facet]) }
```
Comment (david-mears-dfe) [inline config/routes.rb:156]: I think this line should not change.

**Review by david-mears-dfe — APPROVED** [review]
I'm approving in case we decide that fixing the pagination bug is more important than location facets not working properly.

**Review by cpjmcquillan — APPROVED** [review]

**Comment by martin-bangoura** [comment]
Review app deployed to <https://teaching-vacancies-review-pr-2815.london.cloudapps.digital>

**Review by david-mears-dfe — CHANGES_REQUESTED** [review]
On review app, type in a search location 'Oxford' => Sorry, there’s a technical issue at our end

**Review by david-mears-dfe — APPROVED** [review]
cf7 ssh teaching-vacancies-review-pr-2815
~ # cd /teacher-vacancy
/teacher-vacancy # /usr/local/bin/bundle exec rails console
LocationPolygon.count
=> 0

**Pull request merged**